components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/impl/ApplicationsApiServiceImpl.java

```diff
@@ -67,12 +67,20 @@ public class ApplicationsApiServiceImpl implements ApplicationsApiService {
     @Autowired
     private ServerApplicationSharingService applicationSharingService;
 
+    @Deprecated
     @Override
     public Response getAllApplications(Integer limit, Integer offset, String filter, String sortOrder, String sortBy,
                                        String requiredAttributes) {
 
-        ApplicationListResponse listResponse = applicationManagementService
-                .getAllApplications(limit, offset, filter, sortOrder, sortBy, requiredAttributes);
+        return getAllApplications(limit, offset, filter, sortOrder, sortBy, requiredAttributes, true);
```

Comment:
Shouldn't this set as false for excludeSystemPortals?

Reply:
Yes. Addressed with https://github.com/wso2/identity-api-server/pull/711/commits/1db3e049a17d85a6974525c93f50d27520ddb979